CHANGELOG.md

```diff
@@ -5,8 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## [Unreleased]
 
+## [0.2.2] - 2024-03-21
```

Comment:
Unless I'm mistaken, this is only for starky, so the CHANGELOG should reflect it for clarity.